test/cri-containerd/removepodsandbox_test.go

```diff
@@ -0,0 +1,171 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"testing"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+type TestConfig struct {
+	name             string
+	containerName    string
+	requiredFeatures []string
+	runtimeHandler   string
+	sandboxImage     string
+	containerImage   string
+	cmd              []string
+}
+
+func getRunPodSandboxRequest(t *testing.T, runtimeHandler string) runtime.RunPodSandboxRequest {
+	return runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Uid:       "0",
+				Namespace: testNamespace,
+			},
+		},
+		RuntimeHandler: runtimeHandler,
+	}
+}
+
+// Utility function to run tests for a sandbox pod without any containers
```

Comment:
I think this comment should be updated to include more of what the function does. Maybe something like:

```
Utility function to test removal of a sandbox with no containers and no previous call to stop the pod 
```
Or something?